var/spack/repos/builtin/packages/hip/package.py

```diff
@@ -300,7 +306,13 @@ def get_paths(self):
                 msg += "a workaround."
                 raise RuntimeError(msg)
 
+            if "hip" in self.spec.prefix:
```

Comment:
Given that this is a pretty special heuristic I'd write it once, next to the complicated comment, and store the result in some descriptive variable, like `hip_libs_at_top` or something.

Reply:
Done